proto/viam/app/v1/app.proto

```diff
@@ -680,47 +681,38 @@ message DeleteRobotPartSecretRequest {
 message DeleteRobotPartSecretResponse {}
 
 message Authorization {
-  string authorization_type = 1;
-  string authorization_id = 2;
-  string resource_type = 3;
-  string resource_id = 4;
+  string identity_id = 1; 
+  string authorization_type = 2;
+  string authorization_id = 3;
+  string resource_type = 4;
+  string resource_id = 5;
+  string org_id = 6;
```

Comment:
Suggest to keep the numbers of existing fields as is. Just add the new fields with additional numbers. The numbers don't need to be in order. It will cause less backwards incompatible changes.